src/constructs/aws/Queue.ts

```diff
@@ -114,21 +119,27 @@ export class Queue extends AwsConstruct {
 
         const maxRetries = configuration.maxRetries ?? 3;
 
+        const baseName = `${this.provider.stackName}-${id}`;
+
         const dlq = new CdkQueue(this, "Dlq", {
-            queueName: `${this.provider.stackName}-${id}-dlq`,
+            queueName: configuration.fifo === true ? `${baseName}-dlq.fifo` : `${baseName}-dlq`,
             // 14 days is the maximum, we want to keep these messages for as long as possible
             retentionPeriod: Duration.days(14),
+            fifo: configuration.fifo,
+            contentBasedDeduplication: configuration.fifo,
```

Comment:
I don't think the DLQ should deduplicate messages based on content automatically: this is a queue meant to store failed messages over a longer period of time, so we don't want similar messages to be deduplicated.

I'll remove that in the merge process, let me know if you think that's wrong.